packages/asc-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useImperativeHandle } from 'react'
+import useActionOnEscape from '../../utils/hooks/useActionOnEscape'
+import useFocusOnRender from '../../utils/hooks/useFocusOnRender'
+import TextAreaStyle from './TextAreaStyle'
+
+export interface TextAreaMethods
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  setTextAreaRef?: (ref: React.RefObject<HTMLTextAreaElement>) => void
+}
+
+export interface TextAreaProps extends TextAreaMethods {
+  keepFocus?: boolean
+  blurOnEscape?: boolean
+  focusOnRender?: boolean
```

Comment:
These two functionalities have slipped in the `Input` component because we weren't using the `ref` functionality properly. Can you remove them?

Reply:
Done